scripts/ma_dictionaries.py

```diff
@@ -438,6 +438,7 @@
     "ngashore": "NGA shorelines",
     "noaa": "National Oceanic and Atmospheric Administration",
     "npa": "Norwegion Peoples Aid",
+    "ocha": "UN-OCHA 3W Database",
```

Comment:
"UN-OCHA 3W Database" is already registered as "**ocha3w**". let's reuse this code. No need to added once again.